lib/packages/docutils/rstgen.nim

```diff
@@ -26,6 +26,9 @@
 import strutils, os, hashes, strtabs, rstast, rst, highlite, tables, sequtils,
   algorithm, parseutils
 
+when not defined(testing):
```

Comment:
Why is this ``when`` here?

Reply:
In order to remove the dependency on the blockdiag toolchain while testing rstgen.

Reply:
Doesn't this imply others might want to avoid this dependency too?

Reply:
I'm not sure what you mean - anyhow if people run nim doc on files that actually contain blockdiag blocks and the toolchain is not installed a warning is printed and the build still succeed.